docs/glossary.rst

```diff
@@ -62,4 +62,10 @@ Let's go through a few words and their meaning.
         sequential manner. No two jobs with the same lock can run simultaneously. See
         the `howto-locks` section for more information.
 
+    Queueing Lock
+        When configuring a job using `Task.configure` you can attach a queueing lock to
+        the job. No two jobs with the same queueing lock can be waiting in the queue.
+        And `Task.defer` will raise an `AlreadyEnqueued` exception if there already is
+        a job waiting in the queue with the same queueing lock.
```

Comment:
Not sure this sentence belongs in the glossary.

Reply:
Yeah, you're right. I'll remove it.